### ISSUE TITLE
svg image is distorted after copy-paste

### ISSUE THREAD

**Issue by krajcsi** [cogstat/cogstat#225]
The image size is not kept when svg image is copied from CogStat and pasted into LibreOffice Writer on Linux. This is not an issue with the png format, since it is displayed correctly in LibreOffice.

Workaround in LibreOffice Writer: paste the svg image, then copy it from LO, Paste special, and choose SVXB format.

Note that svg image cannot be used in Google Doc because it does not support svg import. Similarly, OnlyOffice (as of 7.2) does not support svg image.

This applies both to the matplotlib-based and the svgutils-based solutions.

**Comment by lakhi** [comment]
do you mean when the svg image it is copied from within the application? for me svg images are not being displayed in the application itself (unlike png images)

**Comment by krajcsi** [comment]
Yes, the image is copied from the CogStat application. I modified the description above accordingly.

Also, if you use a Mac, your issue of svg images not displaying may be related to #249.

**Comment by krajcsi** [comment]
OnlyOffice later version (e.g., 9.4) can handle svg images, and it displays the charts correctly. Proton Docs pastes the svg chart correctly, too. So this may be a LibreOffice-specific issue?
